virttest/remote.py

```diff
@@ -543,19 +545,23 @@ def scp_to_remote(host, port, username, password, local_path, remote_path,
                            "the interface the neighbour attache")
         host = "%s%%%s" % (host, interface)
 
-    command = ("scp -v -o UserKnownHostsFile=/dev/null "
-               "-o StrictHostKeyChecking=no "
-               "-o PreferredAuthentications=password -r %s "
-               "-P %s %s %s@\[%s\]:%s" %
-               (limit, port, quote_path(local_path), username, host,
-                pipes.quote(remote_path)))
+    command = "scp"
+    if directory:
+        command = "%s -r" % command
+    command += (" -v -o UserKnownHostsFile=/dev/null "
+                "-o StrictHostKeyChecking=no "
+                "-o PreferredAuthentications=password %s "
```

Comment:
it looks "limit" argument should be handled like
```
if limit:
    commands += "  -l %s" % limit
```

Reply:
@xutian, it is already handled in this way,

https://github.com/avocado-framework/avocado-vt/blob/edee6694742b60e39a8c7a667185c38fdb039ad7/virttest/remote.py#L519-L538